Sources/Public/BacktraceCrashLoopDetector.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  BacktraceCrashLoopDetector.swift
+//  Backtrace
+//
+
+import Foundation
+
+@objc public class BacktraceCrashLoopDetector: NSObject {
+        
+    internal struct StartUpEvent: Codable {
+        var timestamp: Double
+        var isSuccessful: Bool
+    }
+    
+    @objc private static let plistKey = "CrashLoopDB"
```

Comment:
I don't like this name, since the file contains more than CrashLoops. It contains successful runs as well.

Reply:
Changed to "CrashLoopDetectorData"